src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.ProjectState.cs

```diff
@@ -420,14 +442,14 @@ private Task<StrongBox<ImmutableArray<DiagnosticData>>> DeserializeAsync(Diagnos
                     if (entry.Diagnostics.Length == 0)
                     {
                         // if there is no result, use cached task
-                        return s_emptyResultTaskCache;
+                        return s_emptyResultTaskCache!;
```

Comment:
❕ Avoid this suppression using one of the available alternatives (fix the field definition, or use the `AsNullable` extension)

Reply:
I'm gonna replace using the StrongBox with ValueTask instead.